docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,46 @@
+.. _config_http_filters_cache:
```

Comment:
WDYT about just moving this into a temporary MD/RST file next to the filter source? In general we don't do public docs for things that are not ready to be used yet, and I'm thinking it's not worth it to maintain this doc right now? If we maintain it separately we can finalize the docs when we feel its ready for actual use? (Potentially you can just move the content over to the plugin markdown for now or some other file - up to you)

Reply:
Done in #9972.